packages/demo/src/AppCCP.svelte

```diff
@@ -24,16 +31,22 @@
         });
 
     const measures = [
-        dktkPatientsMeasure,
-        dktkDiagnosisMeasure,
-        dktkSpecimenMeasure,
-        dktkProceduresMeasure,
-        dktkMedicationStatementsMeasure,
-        dktkHistologyMeasure,
+        {
+            name: "DKTK",
```

Comment:
If I understand this correctly is the name associtated with the Measures? In that case it would work to use different measures with a backend. If not then we will have issues, eg. spot needs different measures then spot.

Reply:
The Idea is to have multiple spots with each different measures. the name (here DKTK) is to identify which measure belongs to which spot/blaze